tests/appsec/test_fingerprinting.py

```diff
@@ -106,6 +107,7 @@ def test_session(self):
 
 @rfc("https://docs.google.com/document/d/1DivOa9XsCggmZVzMI57vyxH2_EBJ0-qqIkRHm_sEvSs/edit#heading=h.32nt1jz5tm2n")
 @features.fingerprinting
+@scenarios.remote_config_mocked_backend_asm_dd
```

Comment:
We also need this change for .Net :)

Reply:
I've just rebased and will merge as soon as the CI passes :wink: